ethpm_cli/install.py

```diff
@@ -116,3 +189,9 @@ def update_ethpm_lock(pkg: Package, ethpm_lock: Path) -> None:
     new_pkg_data = pkg.generate_ethpm_lock()
     new_lock = assoc(old_lock, pkg.alias, new_pkg_data)
     ethpm_lock.write_text(f"{json.dumps(new_lock, sort_keys=True, indent=4)}\n")
+
+
+def uninstall_from_ethpm_lock(package_name: str, ethpm_lock: Path) -> None:
+    old_lock = json.loads(ethpm_lock.read_text())
+    new_lock = dissoc(old_lock, package_name)
+    ethpm_lock.write_text(f"{json.dumps(new_lock, sort_keys=True, indent=4)}\n")
```

Comment:
This should probably use a tempfile and then move the file into place.  It's prossible for the `json.dumps` to get interrupted before completing the write to disk at which point the lockfile would be corrupt and unreadable.

Reply:
Is this still necessary if `ethpm_lock` is a [tempdir](https://github.com/ethpm/ethpm-cli/pull/17/files#diff-750d4333bcaf89c20dcf7f7469f3d2baR124)?

Reply:
No, but I don't see anything in this function that requires it to be a temporary directory so I'd be inclined to do it so that it doesn't fail later when someone decides to re-use this function outside of a temporary directory.